cms/management/commands/subcommands/base.py

```diff
@@ -58,18 +62,22 @@ def add_arguments(self, parser):
         self.instances = {}
 
         if self.subcommands:
+            stealth_options = list(self.stealth_options)
             subparsers = parser.add_subparsers(dest=self.subcommand_dest)
             for command, cls in self.subcommands.items():
                 instance = cls(self.stdout._out, self.stderr._out)
                 instance.style = self.style
+                kwargs = {'cmd': self} if DJANGO_2_0 else {}
                 parser_sub = subparsers.add_parser(
-                    cmd=self, name=instance.command_name, help=instance.help_string,
-                    description=instance.help_string
+                    name=instance.command_name, help=instance.help_string,
+                    description=instance.help_string, **kwargs
                 )
 
                 add_builtin_arguments(parser=parser_sub)
                 instance.add_arguments(parser_sub)
+                stealth_options.extend({action.dest for action in parser_sub._actions})
                 self.instances[command] = instance
+            self.stealth_options = tuple(set(stealth_options))
```

Comment:
why is this casted to a set and then a tuple?

Reply:
`set` because I wanted only unique entries and `tuple` to back to original type